tests/operator/operator.go

```diff
@@ -662,23 +661,22 @@ spec:
 			Entry("[test_id:6254] deployments",
 
 				func() {
-
-					vc, err := virtClient.AppsV1().Deployments(originalKv.Namespace).Get(context.Background(), deploymentName, metav1.GetOptions{})
+					patchBytes, err := patch.New(
+						patch.WithAdd(
+							"/spec/template/spec/containers/0/env/-",
```

Comment:
What does `/-` by the end string means?

Reply:
it means "put it at the end of the list"